blindsign/blindrsa/brsa.go

```diff
@@ -98,6 +113,10 @@ func (v randomBRSAVerifier) Hash() hash.Hash {
 	return v.hash
 }
 
+func prepareMsg(message, prefix []byte) []byte {
+	return append(append([]byte{}, prefix...), message...)
```

Comment:
```suggestion
	return append(prefix, message...)
```

is there a specific reason for the append in append? the code above does the same and is more readable. you can check on the [go playground](https://play.golang.com/p/42WXhPQ7BTK).
```go
a := []byte{4, 5, 6}
b := []byte{1, 2, 3}
c := append(b, a...)
d := append(append([]byte{}, b...), a...)

fmt.Printf("a: %v\nb: %v\nc: %v\nd: %v\n", a, b, c, d)
/*
a: [4 5 6]
b: [1 2 3]
c: [1 2 3 4 5 6]
d: [1 2 3 4 5 6]
*/
```